README.md

```diff
@@ -1 +1,135 @@
-# Body-Building
\ No newline at end of file
+<a id="readme-top"></a>
+<div align="center">
+  <h1><b>SUPERPOWER FITNESS CENTER</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [🎥 Loom Video](#Loom-Video)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<br>
+
+# 📖 **SUPERPOWER FITNESS CENTER** <a id="about-project"></a>
+    **SUPERPOWER FITNESS CENTER** is a website that showcases the services and facilities offered by SUPERPOWER FITNESS CENTER, a gym that helps you achieve your fitness goals with personalized training and support. Whether you want to lose weight, gain muscle, or improve your health, this website will help you find the best plan for you.
+<br>
+
+## 🛠 **Built With** <a id="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+<br>
+
+## **Key Features** <a id="key-features"></a>
+- **Ease of use**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# 🚀 **Live Demo** <a id="live-demo"></a>
+- [Click Here](https://chawkimbarki.github.io/Body-Building/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 **Loom Video** <a id="Loom-Video"></a>
+- [Click Here](https://www.loom.com/share/6347120b99d741da855f44bbaae01fa0)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235140317-a4e61a8a-e23e-492e-83ac-3325ecf63c28.png)

- [ ] You have recorded a video of your app :+1: and I enjoyed watching it but I guess you forgot about a requirement for the video which was you need to show a piece of code that you are proud of :thinking: Kindly re-record your video and show that piece of code as well to follow project requirements :+1: